configuration/scripts/options/set_nml.histall

```diff
@@ -237,3 +236,84 @@
     f_aice_ww      = 'md'
     f_diam_ww      = 'md'
     f_hice_ww      = 'md'
+    f_aice             = 'md'
```

Comment:
f_aice is already set above, please remove this entry.

Reply:
looks like f_icedir and f_icespd needs to be added to set_nml.histall.  Should those be in set_nml.cmip?